ast/builtins.go

```diff
@@ -1080,6 +1093,17 @@ var Sprintf = &Builtin{
 	),
 }
 
+// Reverse returns the given string, reversed.
+var Reverse = &Builtin{
+	Name: "reverse",
```

Comment:
I wonder if we should go with `strings.reverse` instead? There's currently only one string function using that prefix, but I think we'd like to avoid using common names as built-ins, plus the `strings.` part would make it obvious what was expected. We currently only do this for `strings.replace_n` though. WDYT @srenatus ?

Reply:
I like the idea, so I went ahead and changed it to what you suggested.
A `reverse(something)` function in a policy would probably require some unnecessary thinking for the readers of said policy, when `strings.reverse` is obvious what it does and what type the argument is/should be.
Also a `reverse` function could maybe lead people to ask "What else does it reverse?", leading to frustration when it only works on strings.